railties/test/application/configuration_test.rb

```diff
@@ -1298,6 +1298,12 @@ def index
       assert ActiveRecord::Base.dump_schema_after_migration
     end
 
+    test "config.active_record.verbose_query_logs is false by default on development" do
```

Comment:
@maclover7 I actually followed the usage in nearby tests but it did bother me so should I take that as an opportunity to fix the language in that whole test file?